datahub-upgrade/src/main/resources/logback.xml

```diff
@@ -0,0 +1,81 @@
+<configuration scan="true" scanPeriod="120 seconds">
+    <property name="LOG_DIR" value="${LOG_DIR:-/tmp/datahub/logs/gms}"/>
```

Comment:
probably should either update this path or ensure that LOG_DIR is set correctly. This is not a blocker since we wouldn't expect these running together with gms

Reply:
done